### ISSUE TITLE
[Bug / Feature] Torrent Autoscrap?`

### ISSUE THREAD

**Issue by DoM1niC** [atmoner/Bittytorrent#9]
If my upload some Torrents, I notice their not Autoscrap to highlight the currently Seeds / Leech, without to manuell Scrap.

**Comment by szunyi** [comment]
Yes this feature is need.
I want to update uploaded torrents with command line script, so with automatism, not manual clicking.

Thanks!

**Comment by atmoner** [comment]
yep.... I can write this but it could take a lot of resources if you have lots of torrents!

**Comment by szunyi** [comment]
lot = lot, thats true :)
but maybe we can do this from VPS, so the main system can be running smooth
